src/controllers/action-queue/types/pushApiTypes.ts

```diff
@@ -100,6 +105,19 @@ export const wasLoginUpdatePayload = uncleaner(asLoginUpdatePayload)
 // Response cleaners
 // ---------------------------------------------------------------------------
 
+const asPushServerResponse =
+  <T>(asT: Cleaner<T>): Cleaner<T> =>
+  raw => {
+    try {
+      const res = asJSON(asEither(asT, asPushError))(raw)
+      if ('error' in res) throw new Error(res.error)
+      return res
+    } catch (err) {
+      console.warn(`${String(err)}:`, raw)
```

Comment:
I'm more agreeable to this as it's used specifically for a push server response which we know should have no keys. The generalized `withConsoleSideEffect` I have to push back on.